data/Vaaran Bhai Gurdas Ji/01.json

```diff
@@ -819,9 +819,9 @@
   },
   {
     "id": "X6A",
-    "sttm_id": 6005,
-    "writer": "Bhai Gurdaas Singh Ji",
-    "section": "Vaars",
+    "sttm_id": 5545,
```

Comment:
How comes the STTM IDs have changed?

Reply:
They haven't changed, rather we had the wrong ID's when we imported.